components/dropdown/src/styles/style.scss

```diff
@@ -32,7 +32,12 @@
 .label {
   font-size: var(--ds-text-body-size-xs, $ds-text-body-size-xs);
   line-height: var(--ds-text-body-size-default, $ds-text-body-size-default);
+  transition: font-size .3s cubic-bezier(.215, .61, .355, 1);
```

Comment:
This code is similar but not exact to what input does. I think we need handle the label styles consistently in both. There is an arguement for extracting the styles controlling the label behavior out of input and dropdown and put it in a shared SCSS file. There is probably other code we can share between the two as well like border behavior, etc.